go/cmd/config/config.go

```diff
@@ -0,0 +1,175 @@
+package config
```

Comment:
I think this should live in the cmd/deploy package, it's only used there,
right now it's in cmd/config, suggesting that `unkey config` is a valid command